docs/design/consistency.md

```diff
@@ -1,21 +1,19 @@
-# Integration
+# Consistency
 
 TigerBeetle is designed to guard against bugs not only in its
 [own code](https://github.com/tigerbeetledb/tigerbeetle/blob/main/docs/TIGER_STYLE.md), but
 at the boundaries, in the application code which interfaces with TigerBeetle.
 This is exhibited by the client's API design, which may be surprising (see [Retries](#retries)) when
 contrasted with a more conventional database.
 
-Strict consistency guarantees (at the database level) simplify application logic and error handling
+Strict consistency guarantees (at the database level) simplify application logic and push error handling
 farther up the stack.
 
-
-
 ## Consistency
```

Comment:
We already have "Consistency" as a h1. Maybe change to "Guarantees" instead?

Reply:
Good idea